docs/DETAILED_INSTALL.md

```diff
@@ -1,8 +1,38 @@
 # Detailed installation guide
 
+### Conda virtual environment
+We recommend using `conda` for package management. These instructions should allow you to install and run mava.
+
+1. Create and activate a virtual environment
+```bash
+conda create -n mava python=3.12
+conda activate mava
+```
+
+2. Clone mava
+```bash
+git clone https://github.com/instadeepai/Mava.git
+cd mava
+```
+
+3. Install the dependencies
+```bash
+pip install -e .
+```
+
+4. Install jax on your accelerator. The example below is for an NVIDIA GPU, please the [official install guide](https://github.com/google/jax#installation) for other accelerators
+```bash
+pip install -U "jax[cuda12]==0.4.25"
+```
+
+5. Run a system!
+```bash
+python mava/systems/ppo/ff_ippo.py env=rware
+```
+
 ### Docker
 
-#### Building the image
+If you are having trouble with dependencies we recommend using our docker image and provided Makefile.
```

Comment:
Are we keeping the docker image the same? (python==3.9 and jax is `jax[cuda11_pip]<=0.4.13`)

Reply:
Can we please update the Dockerfile to also use python 3.12 and the same jax version in the requirements?